src/test/integration_test.ts

```diff
@@ -86,6 +86,35 @@ const fileSpecificIgnoredCodes: {[path: string]: Set<string>} = {
   // https://github.com/PolymerElements/iron-resizable-behavior/pull/25
   'iron-resizable-behavior/test/test-elements.html':
       new Set(['unknown-polymer-behavior']),
+
+  // https://github.com/Polymer/polymer-analyzer/issues/458
+  'hydrolysis/custom_typings/escodegen.d.ts': new Set(['unable-to-analyze']),
```

Comment:
analyzePackage() operates over .d.ts files if there are .d.ts files in the package.

Maybe it shouldn't.